client/onboarding/style.scss

```
@@ -168,4 +168,9 @@ body.wcpay-onboarding__body {
 			color: $gray-700;
 		}
 	}
+
+	// Reposition JITM banners below the fixed onboarding header.
```

Just to make it clearer 😄 
```suggestion
	// Reposition Jetpack's JITM (Just in time messages) banners below the fixed onboarding header.
```